api/routers/auth.js

```diff
@@ -13,18 +13,38 @@ function redirectIfAuthenticated(req, res, next) {
   req.session.authenticated ? res.redirect('/') : next();
 }
 
-function onSuccess(req, res) {
-  req.session.authenticated = true;
-  req.session.authenticatedAt = new Date();
-  req.session.save(() => {
-    res.redirect(req.session.authRedirectPath || '/');
-  });
+// function onSuccess(req, res, next) {
```

Comment:
Do you want to remove this commented out code?

Reply:
yeah, I'm going to push another change to the callback based on your other questions/feedback